packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -0,0 +1,303 @@
+import {LightningElement, api, track, wire} from 'lwc';
+import caseClassificationTitle from '@salesforce/label/c.quantic_CaseClassificationTitle';
+import moreTopics from '@salesforce/label/c.quantic_MoreTopics';
+import selectOption from '@salesforce/label/c.quantic_SelectOption';
+import loading from '@salesforce/label/c.quantic_Loading';
+import {
+  getObjectInfo,
+  getPicklistValuesByRecordType,
+} from 'lightning/uiObjectInfoApi';
+// @ts-ignore
+import CASE_OBJECT from '@salesforce/schema/Case';
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+} from 'c/quanticHeadlessLoader';
+
+/** @typedef {import("coveo").CaseAssistEngine} CaseAssistEngine */
+/** @typedef {import("coveo").CaseField} CaseField */
+
+/**
+ * The `QuanticCaseClassification` component displays field value suggestions returned by Coveo Case Assist as well as a single-select dropdown containing other available values.
```

Comment:
We should add another sentence saying these other values in the dropdown are taken from the org's CASE_OBJECT record type.